### PR TITLE
chore: add coverage report to PR

### LINKED ISSUE #224: Show test coverage report to PR Comment
### Description

In creating the test code.
I want the coverage to be automatically commented on the GitHub PR.

### Problem Statement/Justification

when reviewing by myself / requesting PR review
It is tedious to check coverage when reviewing/asking for PRs

### Proposed Solution or API

Automatic commenting by GitHub Actions

reference:
https://github.com/marketplace/actions/jest-coverage-comment

### Alternatives

_No response_

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by matsu3m**
Closes #224 

## Description

It seems the issue mentioned above was left unaddressed, so I handled it. With this fix, an overall coverage report will be displayed, and annotations will be added to branches that are not covered. Please feel free to close this PR if it's unnecessary. Thank you.

image
image

## Current behavior (updates)

No Coverage report.

## New behavior

Coverage report will be displayed on PR.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Comment by matsu3m** [comment]
It seems that the tests are failing to execute. I apologize for the oversight. It was working fine in the forked repository, but...

Regarding `HttpError: Resource not accessible by integration`, it's highly likely that this (https://github.com/ArtiomTr/jest-coverage-report-action?tab=readme-ov-file#forks-with-no-write-permission) is the cause.

As for  `ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL  Command "test:ci" not found`, I wonder what that could be... Do you have any idea?

**Comment by hirotomoyamada** [comment]
@matsu3m 

Hi, thanks for the PR!

Sorry. Currently I'm fully committed to v1.2.0.

I would like to investigate this PR once it is released.

**Comment by matsu3m** [comment]
@hirotomoyamada 
OK, I believe this PR is of low priority, so please prioritize your work on v1.2.0. I will also try to investigate the cause when I have time. Thank you.

File: .github/workflows/quality.yml [hunk 1]
```diff
@@ -30,7 +30,10 @@ jobs:
         uses: ./.github/composite-actions/install
 
       - name: Run tests
-        run: pnpm test
+        uses: ArtiomTr/jest-coverage-report-action@v2
+        with:
+          package-manager: pnpm
+          test-script: pnpm test:ci
```
Comment (illionillion) [inline .github/workflows/quality.yml:36]: Try fix it.

`npx jest --coverage --silent --testLocationInResults --ci --json --outputFile=report.json`

Comment (illionillion) [reply .github/workflows/quality.yml:36]: See this PR.

https://github.com/hirotomoyamada/yamada-ui/pull/683

Comment (hirotomoyamada) [reply .github/workflows/quality.yml:36]: I would like to try various GitHub Actions, so I'll merge this PR for now.

**Comment by hirotomoyamada** [comment]
@matsu3m @illionillion 

Hi, I would like to try various GitHub Actions, so I'll merge this PR for now.

**Pull request merged**